.github/workflows/test_rulesets.yml

```diff
@@ -0,0 +1,46 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test-rulesets:
+    name: runner / tflint (rulesets)
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ./
+        continue-on-error: true
+        id: test
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-check
+          level: info
+          tflint_rulesets: "azurerm google"
+          working_directory: testdata_rulesets
+
+      # The check is expected to fail on the test data
+      - name: Check return codes
+        if: success() || failure ()
```

Comment:
Maybe:
```
if: !cancelled()
```

it's slightly easier to understand than guessing what it's actually doing.

Reply:
To me, that looks even more confusing: what does `canceled()` even mean (manual cancel? error?)? 

Using `success()` and `failure()` makes it a bit clearer that "_hey, run this step all the time_".